x-pack/plugins/observability_solution/observability_ai_assistant_app/server/types.ts

```diff
@@ -22,9 +46,27 @@ export interface ObservabilityAIAssistantAppServerSetup {}
 export interface ObservabilityAIAssistantAppPluginStartDependencies {
   observabilityAIAssistant: ObservabilityAIAssistantServerStart;
   ruleRegistry: RuleRegistryPluginStartContract;
+  alerting: AlertingPluginStart;
+  licensing: LicensingPluginStart;
+  actions: ActionsPluginStart;
+  security: SecurityPluginStart;
+  features: FeaturesPluginStart;
+  taskManager: TaskManagerStartContract;
+  dataViews: DataViewsServerPluginStart;
+  cloud?: CloudStart;
+  serverless?: ServerlessPluginStart;
 }
 
 export interface ObservabilityAIAssistantAppPluginSetupDependencies {
   observabilityAIAssistant: ObservabilityAIAssistantServerSetup;
   ruleRegistry: RuleRegistryPluginSetupContract;
+  alerting: AlertingPluginSetup;
+  licensing: LicensingPluginSetup;
+  actions: ActionsPluginSetup;
+  security: SecurityPluginSetup;
+  features: FeaturesPluginSetup;
+  taskManager: TaskManagerSetupContract;
+  dataViews: DataViewsServerPluginSetup;
+  cloud?: CloudSetup;
+  serverless?: ServerlessPluginSetup;
```

Comment:
Were all of these just missing before? I don't see you adding so many deps in kibana.json

Reply:
Yes they were not used so I think we didn't have to define them in the contract